doc/DeveloperGuide.md

```diff
@@ -88,6 +88,44 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+
+#### Use case: Rename the tag 'friends' into 'buddies'
```

Comment:
This use case is too specific, you can change it to something like `Rename tags`.